resources/ts/Pages/Integrations/Index.tsx

```diff
@@ -22,6 +22,7 @@ import { IntegrationType } from "../../types/IntegrationType";
 import { Values } from "../../types/Values";
 import { Auth0Tenant } from "../../types/Auth0Tenant";
 import { UiTiDv1Environment } from "../../types/UiTiDv1Environment";
+import { KeyVisibility } from "../../types/KeyVisibility";
```

Comment:
I think here you want to import the type?
```suggestion
import type { KeyVisibility } from "../../types/KeyVisibility";
```

Reply:
I will create a PR to add this eslitn rule.
This way we don't have to worry about it.

https://typescript-eslint.io/rules/consistent-type-imports/